tests/Hybrid/CreateTable/MiscCreateTest.php

```diff
@@ -54,7 +54,7 @@ public function all_keys_are_added_in_create_rowstore()
     /** @test */
     public function fulltext_index()
     {
-        if (version_compare(Application::VERSION, '8.12.0', '<=')) {
+        if (version_compare(Application::VERSION, '8.79.0', '<=')) {
```

Comment:
same thing here

Reply:
actually - you can just remove fulltext from these tests since your new full text tests are covering

Reply:
You're right, yeah